### ISSUE TITLE
debian base-image build for EKS Go

### ISSUE THREAD

**Issue by danbudris** [aws/eks-distro-build-tooling#669]
We want a basic build for a Debian-based image which contains our EKS Go version as the primary Go version.

We should have an image tag per supported Go version in public ECR. 

This should be built and pushed during the prod release for EKS Go.

**Comment by danbudris** [comment]
see https://github.com/aws/eks-distro-build-tooling/pull/667

**Comment by danbudris** [comment]
https://github.com/aws/eks-distro-build-tooling/pull/667 is merged! 

we'll probably have to make some tweaks based on post-submit behavior, but we're well on our way to having the build needed for EKS to build the upstream K8s toolchain images like kube-cross w/an EKS Go base-image.

**Comment by danbudris** [comment]
post-submit behavior was adjusted to include a new image-build post-submit; doing the image-build int he same post-submit job as the original wasn't tenable given the way we have our credentials set up.

Once that got figured out, and we added a new trigger for the debian base image, everything ran as expected and pushed to public ECR. I pulled down the images and smoke-tested, and they looked OK.

I let Ernest know, and now we're working on getting the latest 1.18 and 1.19 out and into the images. But I'm going to close thise ticket -- as we have the buidl system, and we have the images in public ECR handed-off
